### DISCUSSION TITLE
OpenGlobus logo

### DISCUSSION THREAD

**Discussion by busatoa** [openglobus/openglobus#803] [Q&A] [answered]
Hi,
I'd like to mention openglobus into my webapp in the "powered-by" section,
is there any openglobus logo?

Best regards,
Alessandro

**Comment by Zemledelec** [comment, marked as answer]
Hey @busatoa ! You can use the repository logo https://avatars.githubusercontent.com/u/2453835?s=400&u=f7d9560971b43f389d2b9411cded4a1ceb82d31b&v=4

It won't be changed for a while.

**Comment by Zemledelec** [comment]
Could you share your project URL, please? I'm going to add a section in the openglobus homepage like `Trusted by`.

**Reply by busatoa** [reply]
https://snapglobe.ogs.it
The objective of this project is a first step to become an entry point to search and access georeferenced dataset for various associated partners, the search leads to a link to access the portal of the partner who owns the dataset.

**Reply by Zemledelec** [reply]
This is great @busatoa ! Can I also make a post about Snapglobe in linkedin and tweeter openglobus news?

**Reply by busatoa** [reply]
yes, you can post it. Thank you.
